### PR TITLE
support mask_rcnn for kunlun

### PR DESCRIPTION
**Pull request by QingshuChen**
support mask_rcnn for kunlun

### CODE REVIEW

File: configs/mask_rcnn_r50_1x_kunlun.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+architecture: MaskRCNN
+use_gpu: false
+use_xpu: true
+max_iters: 2400
+snapshot_iter: 50
+log_iter: 1
+save_dir: output
+pretrain_weights: https://paddle-imagenet-models-name.bj.bcebos.com/ResNet50_cos_pretrained.tar
+metric: COCO
+weights: output/mask_rcnn_r50_1x/model_final
```
Comment (jerrywgz) [inline configs/mask_rcnn_r50_1x_kunlun.yml:10]: output/mask_rcnn_r50_1x_kunlun/model_final

**Comment by jerrywgz** [comment]
pre-commit is needed

File: configs/mask_rcnn_r50_1x_kunlun.yml (hunk 1 above)
Comment (QingshuChen) [reply configs/mask_rcnn_r50_1x_kunlun.yml]: done

**Comment by QingshuChen** [comment]
> pre-commit is needed

有什么wiki吗？如何使用这个pre-commit?

**Pull request closed without merging**